### PR TITLE
Fixed income unit tests

### PR DESCRIPTION
**Pull request by IgorWounds**
# Description

- [ ] Summary of the change / bug fix.
- [ ] Link # issue, if applicable.
- [ ] Screenshot of the feature or the bug before/after fix, if applicable.
- [ ] Relevant motivation and context.
- [ ] List any dependencies that are required for this change.

# How has this been tested?

* Please describe the tests that you ran to verify your changes.
* Provide instructions so we can reproduce.
* Please also list any relevant details for your test configuration.
- [ ] Make sure affected commands still run in terminal
- [ ] Ensure the SDK still works
- [ ] Check any related reports

# Checklist:

- [ ] I have adhered to the GitFlow naming convention and my branch name is in the format of `feature/feature-name` or `hotfix/hotfix-name`.
- [ ] Update [our documentation](https://openbb-finance.github.io/OpenBBTerminal/) following [these guidelines](https://github.com/OpenBB-finance/OpenBBTerminal/tree/main/website).  Update any user guides that are affected by the changes.
- [ ] Update our tests following [these guidelines](https://github.com/OpenBB-finance/OpenBBTerminal/tree/main/tests).
- [ ] Make sure you are following our [CONTRIBUTING guidelines](https://github.com/OpenBB-finance/OpenBBTerminal/blob/main/CONTRIBUTING.md).
- [ ] If a feature was added make sure to add it to the corresponding [integration test script](https://github.com/OpenBB-finance/OpenBBTerminal/tree/develop/openbb_terminal/miscellaneous/integration_tests_scripts).

# Others
- [ ] I have performed a self-review of my own code.
- [ ] I have commented my code, particularly in hard-to-understand areas.

### CODE REVIEW

**Comment by jmaslek** [comment]
FYI your fred api key is in the requests.

File: tests/openbb_terminal/fixedincome/record/http/test_fred_model/test_get_ameribor[overnight-2020-01-01-2020-02-01].yaml [hunk 1]
```diff
@@ -0,0 +1,85 @@
...
+    body: null
+    headers:
+      Connection:
+      - close
+      Host:
+      - api.stlouisfed.org
+      User-Agent:
+      - Python-urllib/3.9
+    method: GET
+    uri: https://api.stlouisfed.org/fred/series/observations?series_id=AMERIBOR&observation_start=2020-01-01&observation_end=2020-02-01&api_key=e59724f37e9be8056dccc2053fda6341
```
Comment (jmaslek) [inline tests/openbb_terminal/fixedincome/record/http/test_fred_model/test_get_ameribor[overnight-2020-01-01-2020-02-01].yaml:12]: rip

**Comment by jmaslek** [comment]
Im also not sure why the unit tests aren't running.  theres a changed .py file sooooooo @luqmanbello ?

**Comment by luqmanbello** [comment]
> Im also not sure why the unit tests aren't running. theres a changed .py file sooooooo @luqmanbello ?

It was a seriously weird issue, but I can see it's running now

**Comment by Chavithra** [comment]
> Im also not sure why the unit tests aren't running. theres a changed .py file sooooooo @luqmanbello ?

It shouldn't be only python files, we have cassette and other types.

Maybe a good rule can be:
- openbb_terminal/*
- tests/*
- pyproject.toml, poetry.lock, requirements.txt, requirements-full.txt

**Review by Chavithra — APPROVED** [review]
Looks ok to me.
The tests are running fine on my side.

**Pull request merged**